styles.css

```diff
@@ -0,0 +1,576 @@
+section {
+  display: flex;
+  margin: auto;
+  width: 375px;
+  position: absolute;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 70px 24px 144px 24px;
+  gap: 5px;
+  border-bottom-left-radius: 40px;
+  top: 82.8px;
+  left: calc(50% - (187.5px));
+  overflow: hidden;
+  background: #fff;
+}
+
+i {
+  position: absolute;
+  left: 83.67%;
+  right: 16.67%;
+  top: 29.17%;
+  bottom: 70.83%;
+}
+
+.container {
+  z-index: 1;
+  display: flex;
+  flex-direction: column;
+  height: 211px;
+  width: 327px;
+  left: 27px;
+  top: 156px;
+  border-radius: null;
+}
+
+ul {
+  display: flex;
+  justify-content: center;
+  order: 3;
+  gap: 15px;
+  margin-left: 25%;
+}
+
+li {
+  list-style-type: none;
+  height: 32px;
+  display: flex;
+  justify-content: center;
+}
+
+h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700px;
+  font-size: 40px;
+  line-height: 108%;
+  text-align: center;
+  display: flex;
+  color: #172b4d;
+  justify-content: center;
+}
+
+section p {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #172b4d;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  text-align: center;
+  margin-top: -10%;
+}
+
+section h3 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 117px;
+  height: 24px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.green {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700px;
+  font-size: 40px;
+  line-height: 130%;
+  text-align: center;
+  display: flex;
+  color: #36b37e;
+  justify-content: center;
+}
+
+.menu_main .toggler {
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: auto;
+  z-index: 4;
+  cursor: pointer;
+  width: 50px;
+  height: 50px;
+  opacity: 0;
+}
+
+.menu_main .humbuger {
+  position: absolute;
+  right: 0;
+  top: 0;
+  z-index: 3;
+  width: 60px;
+  height: 60px;
+  padding: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+/* .humbuger line */
+
+.menu_main .humbuger > div {
+  position: relative;
+  width: 100%;
+  height: 1px;
+  background-color: blue;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition: all 0.4s ease;
+}
+
+/* Top and bottom line */
+.menu_main .humbuger > div::after,
+.menu_main .humbuger > div::before {
+  content: "";
+  position: absolute;
+  top: 0.25em;
+  left: 0;
+  width: 1em;
+  height: 0.125em;
+  border-top: 0.375em double #000;
+  border-bottom: 0.125em solid #000;
+}
+
+/* moves lin down */
+.menu_main .humbuger > div::after {
+  top: 10px;
+  width: 100%;
+}
+
+.menu_main .menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  margin: auto;
+  margin-left: 10%;
+  margin-top: 2%;
+  height: 100%;
+  width: 100%;
+  border-radius: 0;
+  overflow: hidden;
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  visibility: unset;
+}
+
+.overlay ::before {
+  content: "";
```

Comment:
Kindly implement the following changes to enable your design to look better and make it look close as possible to the design given :

- [ ] Kindly increase the size of the hamburger menu so it looks close to the design 😄 
- [ ] please add the background image close to the hamburger menu to have the design look similar to the template given 👍 
 

Your design :
![image](https://github.com/tsigewossne/portfolio/assets/20537712/d3034928-c16b-4f8e-8519-6853419cd849)

Figma template :

![image](https://github.com/tsigewossne/portfolio/assets/20537712/127a682c-3189-49c8-b355-f4a2f4d5b3fb)